It seems you're almost there: only the flake8 tests seem to be failing:

```
check runtests: commands[2] | flake8 src
src/pyejabberd/client.py:364:161: E501 line too long (163 > 160 characters)
src/pyejabberd/client.py:396:5: E303 too many blank lines (2)
src/pyejabberd/definitions.py:200:77: W291 trailing whitespace
src/pyejabberd/definitions.py:201:67: W291 trailing whitespace
src/pyejabberd/definitions.py:210:77: W291 trailing whitespace
src/pyejabberd/definitions.py:226:24: E231 missing whitespace after ','
ERROR: InvocationError: '/home/ubuntu/pyejabberd/.tox/check/bin/flake8 src'
```

--- a/src/pyejabberd/client.py
+++ b/src/pyejabberd/client.py
@@ -361,7 +361,10 @@ class EjabberdAPIClient(contract.EjabberdAPIContract):
         :type subs: str|unicode
         :return:
         """
-        return self._call_api(definitions.AddRosterItem, localuser=localuser, localserver=localserver, user=user, server=server, nick=nick, group=group, subs=subs)
+        return self._call_api(definitions.AddRosterItem,
+                              localuser=localuser, localserver=localserver,
+                              user=user, server=server,
+                              nick=nick, group=group, subs=subs)
 
     def delete_rosteritem(self, localuser, localserver, user, server):
         """
@@ -391,7 +394,6 @@ class EjabberdAPIClient(contract.EjabberdAPIContract):
         :return: A list of user's contacts
         """
         return self._call_api(definitions.GetRoster, user=user, host=host)
-
 
     def _validate_and_serialize_arguments(self, api, arguments):
         """

--- a/src/pyejabberd/definitions.py
+++ b/src/pyejabberd/definitions.py
@@ -197,8 +197,8 @@ class GetRoomAffiliations(API):
 
 class AddRosterItem(API):
     method = 'add_rosteritem'
-    arguments = [StringArgument('localuser'), StringArgument('localserver'), 
-                 StringArgument('user'), StringArgument('server'), 
+    arguments = [StringArgument('localuser'), StringArgument('localserver'),
+                 StringArgument('user'), StringArgument('server'),
                  StringArgument('nick'), StringArgument('group'), StringArgument('subs')]
 
     def transform_response(self, api, arguments, response):
@@ -207,7 +207,7 @@ class AddRosterItem(API):
 
 class DeleteRosterItem(API):
     method = 'delete_rosteritem'
-    arguments = [StringArgument('localuser'), StringArgument('localserver'), 
+    arguments = [StringArgument('localuser'), StringArgument('localserver'),
                  StringArgument('user'), StringArgument('server')]
 
     def transform_response(self, api, arguments, response):
@@ -223,7 +223,7 @@ class GetRoster(API):
         for contact in response.get('contacts', []):
             contact_details = {}
             for parameter in contact['contact']:
-                for key,value in parameter.items():
+                for key, value in parameter.items():
                     contact_details[key] = value
             roster.append(contact_details)
         return roster

